### ISSUE TITLE
Time : No provider configured

### ISSUE THREAD

**Issue by NebzHB** [mfucci/node-matter#90]
Hi,

wonderfull project !!! thank you !

I was interested to test that early dev but i receive an error : 
```
Nebz@testSystem:~/node-matter$ matter
/usr/lib/node_modules/node-matter/build/time/Time.js:13
Time.get = () => { throw new Error("No provider configured"); };
                   ^

Error: No provider configured
    at Function.Time.get (/usr/lib/node_modules/node-matter/build/time/Time.js:13:26)
    at new Logger (/usr/lib/node_modules/node-matter/build/log/Logger.js:46:33)
    at Function.get (/usr/lib/node_modules/node-matter/build/log/Logger.js:55:16)
    at Object.<anonymous> (/usr/lib/node_modules/node-matter/build/net/node/UdpChannelNode.js:18:32)
    at Module._compile (node:internal/modules/cjs/loader:1126:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1180:10)
    at Module.load (node:internal/modules/cjs/loader:1004:32)
    at Function.Module._load (node:internal/modules/cjs/loader:839:12)
    at Module.require (node:internal/modules/cjs/loader:1028:19)
    at require (node:internal/modules/cjs/helpers:102:18)

```

did i missed something ?

as it might be important : 

```
Nebz@testSystem:~/node-matter$ locale
LANG=fr_FR.UTF-8
LANGUAGE=
LC_CTYPE="fr_FR.UTF-8"
LC_NUMERIC="fr_FR.UTF-8"
LC_TIME="fr_FR.UTF-8"
LC_COLLATE="fr_FR.UTF-8"
LC_MONETARY="fr_FR.UTF-8"
LC_MESSAGES="fr_FR.UTF-8"
LC_PAPER="fr_FR.UTF-8"
LC_NAME="fr_FR.UTF-8"
LC_ADDRESS="fr_FR.UTF-8"
LC_TELEPHONE="fr_FR.UTF-8"
LC_MEASUREMENT="fr_FR.UTF-8"
LC_IDENTIFICATION="fr_FR.UTF-8"
LC_ALL=
```

**Comment by mfucci** [comment]
Thanks Nebz!

Seems like my latest release was a bad one.
Hold tight, I will have it fixed shortly...

**Comment by mfucci** [comment]
It was a bug in package.json: I renamed the main file and I forgot to update package.json to have the "matter" command link to the right file. Also the release was containing the compiled version of the old file, creating the bug that you are seeing.

Fixed now with v0.0.10 just published
Thanks for reporting this!
